pkg/config/apm.go

```diff
@@ -134,6 +130,21 @@ func setupAPM(config Config) {
 	})
 }
 
+func parseKVList(key string) func(string) interface{} {
+	return func(in string) interface{} {
+		if len(in) == 0 || in[0] != '[' {
+			return strings.Split(in, " ")
+		}
```

Comment:
Although clever, I find it a bit confusing. I'd prefer being more verbose here:
```suggestion
		if len(in) == 0 {
			return []string{}
		}
		if in[0] != '[' {
			return strings.Split(in, " ")
		}
```
WDYT?